hdl/syn/bit_to_mcs_to_svf.cmd

```diff
@@ -0,0 +1,39 @@
+write_cfgmem -force -format mcs -size 32 -interface SPIx4 -loadbit {up 0x00000000 SYN_PROJECT.bit } -file SYN_PROJECT.mcs
+
+open_hw
+current_hw_target [get_hw_targets */xilinx_tcf/Xilinx/nat-mch-dighomolog:2542]
```

Comment:
Genuine question: is this line necessary for Vivado to generate SVF files? It seems a little awkward to specify a hardware target when we aren't connecting to it in this case.